rita/src/rita_client/dashboard/wifi.rs

```diff
@@ -93,3 +96,51 @@ impl Handler<WifiPass> for Dashboard {
         Ok(())
     }
 }
+
+pub struct GetWifiConfig;
+
+impl Message for GetWifiConfig {
+    type Result = Result<Vec<WifiInterface>, Error>;
+}
+
+impl Handler<GetWifiConfig> for Dashboard {
```

Comment:
Why exactly does this need to happen in the Dashboard actor? This message would generally clutter `Dashboard`'s mailbox while the data could be perfectly obtained from the request handler

Reply:
it's this or we put everything into the network endpoints receiver functions.  Might be better, but not by much.